ntbs-service/Services/AuditService.cs

```diff
@@ -14,19 +14,15 @@ public class AuditService : IAuditService
         private AuditDatabaseContext auditContext;
 
         private const string READ_EVENT = "Read";
-        private bool shouldAudit;
 
-        public AuditService(AuditDatabaseContext auditContext, IConfiguration configuration)
+        public AuditService(AuditDatabaseContext auditContext)

```

Comment:
Now just checking in startup whether to add auditing in the middleware, which makes more sense.

Reply:
nice!